lib/check_search.rb

```diff
@@ -172,7 +172,7 @@ def should_hit_elasticsearch?
   end
 
   def media_types_filter
-    MEDIA_TYPES & @options['show']
+    [MEDIA_TYPES, 'blank'].flatten & @options['show']
```

Comment:
I expected in this case to add `blank` to MEDIA_TYPES array why not?

Reply:
@melsawy we want to allow to filter by `blank` but not include it as a default media type, as a way to keep moving towards deprecating it.

Reply:
OK @caiosba